### DISCUSSION TITLE
Is this the real codebase?

### DISCUSSION THREAD

**Discussion by spiicy-sauce** [nickcoutsos/keymap-editor#168] [General]
👋 Hello there. I'm a FE developer and (relatively new) keyboard hobbyist. I was poking around the source code and noticed some pretty big differences in the code on `main` and the app in production. Has the source code been moved to a private repository?

**Comment by nickcoutsos** [comment]
Hello. You are correct, the public repo doesn't contain the current app source. You can read more here: https://github.com/nickcoutsos/keymap-editor/wiki/Source-Code-Updates

**Comment by spiicy-sauce** [comment]
OH man that's a bummer, but totally makes sense. It's lame to hear that companies were using it for their products without giving back to the open-source editor.

Well, if you are ever interested in bringing on more collaborators, I'm a long-time product engineer with tons of experience in design and UX engineering (mostly React, TS stuff). I'm stoked on building some new boards and will be using the keymap editor to program them, and would love to contribute in some way if you're open to it, but I also understand if you're not interested.

It's a really impressive app and I've been enjoying using it so far.

**Reply by nickcoutsos** [reply]
Thanks for the kind words and interest, for now it's enough just to know that it's useful :)
